### ISSUE TITLE
Accessibility for numeronyms

### ISSUE THREAD

**Issue by jacobwarduk** [a11yproject/a11yproject.com#507]
Referencing Issue #504, I believe we need an agreed upon way of addressing numeronyms.

They differ from acronyms or abbreviations in that they "do not look like normal words and cannot be pronounced according to the usual rules of the language" [[Specific Benefits of Success Criterion 3.1.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-located.html)]. This is evidenced by the fact that people pronounce "a11y" as "Alley", "A-eleven-Y", "A-1-1-Y".

Since screen readers generally don't read the `title` attribute of `<abbr />` tags unless they are navigable, the [proposed solutions by W3C to meet Level AAA](https://www.w3.org/WAI/WCAG20/quickref/20160105/#meaning-located) are:

1. [G97: Providing the first use of an abbreviation immediately before or after the expanded form](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G97)

2. [G55: Linking to definitions](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/G55)

**Comment by svinkle** [comment]
Closing via #518.
